bindings/python/requirements.txt

```
@@ -1,3 +1,6 @@
 # Apache License 2.0
 
-open-space-toolkit-astrodynamics~=9.0.3
+open-space-toolkit-core~=3.0
+open-space-toolkit-mathematics~=3.0
```

Shouldn't we add io as well here since its defined in the Cmake?
```suggestion
open-space-toolkit-io~=3.0
open-space-toolkit-mathematics~=3.0
```

---

Yeah I'm a bit confused on this one. As it's not used by OSTk Simulation directly, it's used by OSTk physics. But since we do shared libs of this, we could end up in a state where Physics updates to OSTk IO 4.0 and Simulation doesn't and therefore it's in a bad state.

---

I will make the corresponding change to OSTk Astro as well.